app/controllers/user.js

```diff
@@ -2,22 +2,13 @@ const User = require('../models').users,
   errors = require('../errors'),
   logger = require('../logger'),
   jwt = require('jwt-simple'),
-  secret = process.env.NODE_API_JWT_SECRET_STRING,
-  { check, validationResult } = require('express-validator/check'),
+  config = require('../../config/index'),
```

Comment:
When the file you are requiring is called index, you can avoid the name and just use `config = require ('../../config),`